src/rgw/rgw_sal.h

```diff
@@ -383,7 +383,7 @@ class RGWObject {
     virtual RGWAccessControlPolicy& get_acl(void) = 0;
     virtual int set_acl(const RGWAccessControlPolicy& acl) = 0;
     virtual void set_atomic(RGWObjectCtx *rctx) const = 0;
-    virtual void set_prefetch_data(RGWObjectCtx *rctx) = 0;
+    virtual void set_prefetch_data(RGWObjectCtx *rctx, prefetch_range *range = nullptr) = 0;
```

Comment:
can we please remove this from the RGWObject interface, and call into RGWObjectCtx directly?